src/components/portfolio/PortfolioOverview.test.tsx

```diff
@@ -33,22 +35,20 @@ describe('PortfolioOverview', () => {
   });
 
   test('switch between table and cards view', () => {
-    // TODO: Update test once cards view is implemented
     const { container, queryByText } = renderComponent();
     expect(queryByText('portfolio.dashboard.title')).toBeInTheDocument();
     expect(queryByText('portfolio.dashboard.noPortfolios')).toBeNull();
-    expect(queryByText('cards')).toBeNull();
```

Comment:
There should be a check that the cards view is not visible: `expect(container.querySelectorAll('ul')).toHaveLength(0);`